### PR TITLE
test: add logger for e2e test framework

### PR DESCRIPTION
**Pull request by huoqifeng**
Fixes: #726

### CODE REVIEW

**Review by liudalibj — APPROVED** [review]
LGTM

File: test/e2e/main_test.go [hunk 1]
```diff
@@ -26,7 +27,7 @@ func TestMain(m *testing.M) {
 	// CLOUD_PROVIDER is required.
 	cloudProvider = os.Getenv("CLOUD_PROVIDER")
 	if cloudProvider == "" {
-		fmt.Println("CLOUD_PROVIDER should be exported in the environment")
+		log.Info("CLOUD_PROVIDER should be exported in the environment")
```
[the remark this answers is not in the document]
Comment (huoqifeng) [reply test/e2e/main_test.go:30]: Agree, thanks! @stevenhorsman

File: cmd/cluster-provisioner/main.go [hunk 2]
```diff
@@ -29,71 +30,71 @@ func main() {
...
 			os.Exit(1)
 		}
 	}
 
 	if *action == "deprovision" {
-		fmt.Println("Deleting Cluster...")
+		log.Info("Deleting Cluster...")
 		if err := provisioner.DeleteCluster(context.TODO(), cfg); err != nil {
-			fmt.Println(err)
+			log.Info(err)
```
[the remark this answers is not in the document]
Comment (stevenhorsman) [reply cmd/cluster-provisioner/main.go]: You've addressed my concern, which was logging the error as an info message. Thanks!

File: cmd/cluster-provisioner/main.go [hunk 3]
```diff
@@ -29,71 +30,71 @@ func main() {
 
 	provisioner, err := pv.GetCloudProvisioner(cloudProvider, provisionPropsFile)
 	if err != nil {
-		fmt.Println(err)
+		log.Error(err)
 		os.Exit(1)
```
Comment (yoheiueda) [inline cmd/cluster-provisioner/main.go:34]: FYI. Logrus looks like providing `log.Fatal` for this code pattern.
https://pkg.go.dev/github.com/sirupsen/logrus#Fatal

Comment (huoqifeng) [reply cmd/cluster-provisioner/main.go:34]: @yoheiueda yes, Fatal is right approach here, I'll fix it. Thank you!

**Pull request merged**